package.json

```diff
@@ -96,14 +96,14 @@
     "eslint-plugin-unused-imports": "^2.0.0",
     "execa": "^5.0.0",
     "glob": "^8.0.1",
-    "jest": "^27.2.3",
+    "jest": "^28.0.1",
     "jest-junit": "^13.0.0",
     "jsdom": "^19.0.0",
     "lerna": "^4.0.0",
     "license-checker": "^25.0.1",
     "madge": "^5.0.0",
     "prettier": "^2.0.2",
-    "ts-jest": "^27.0.5",
+    "ts-jest": "^28.0.0-next.1",
```

Comment:
I'm fine with the workaround due to the missing `@types/jest`, as far as we have a follow up.
However, this is also an unstable version.
I feel we can wait a bit if 2 libs are not ready.

Reply:
>The reason was that jest 28 entered the project.

`jest` is not used as direct dependency of `express`.
Could you please share a bit more?

Reply:
I would also merge it but perhaps is this too unstable. Other opinions?

Reply:
I am a bit confused, because I don't recognize how jest is related to express. If it's not I would prefer to have this on separate PRs.
That said, I think we should just ask dependabot to open a new PR when there is a new version of the types and close those related to jest until then. That way, we don't have to remember removing the "any" later on.

Reply:
That sounds like a good plan, is there a dedicated way to say to dependdabot please watch for @types 28? Regarding the jest version I am also confused why the jest expect was updated in the yarn lock file 🤷‍♂️. I removed it and recreated it with the express version updated and somewho this enters via express. Perhaps express contains some test scafolding?